static/js/types/websites.ts

```diff
@@ -38,4 +38,18 @@ export interface Website {
   source: string | null
   starter: WebsiteStarter | null
   metadata: any
+  is_admin?: boolean // eslint-disable-line
+}
+
+export interface WebsiteCollaborator {
+  role: string
```

Comment:
I added the new type and used it for `WebsiteCollaboratorFormData` but not `WebsiteCollaborator` because the latter could also be one of `ROLE_GLOBAL` and `ROLE_OWNER`, neither of which are allowed for creating/editing collaborators.

Reply:
Added 2 types, `WebsiteRole` for `WebsiteCollaborator` and `WebsiteRoleEditable` for `WebsiteCollaboratorFormData`